### DISCUSSION TITLE
paper/%EC%84%A0%ED%98%95-%EB%8C%80%EC%88%98%ED%95%99

### DISCUSSION THREAD

**Discussion by giscus** [ysw421/blog#11] [General]
# paper/%EC%84%A0%ED%98%95-%EB%8C%80%EC%88%98%ED%95%99

안녕하세요, 환영합니다!

https://www.siwonsw.com/paper/%EC%84%A0%ED%98%95-%EB%8C%80%EC%88%98%ED%95%99

**Comment by dongkyu12345** [comment]
시원님 멋있어요

**Comment by sonminjun66** [comment]
와 개쩐다

**Comment by swimoneshot** [comment]
최고에요 시원님

**Comment by 0720al** [comment]
시원님 저도 시원님처럼 멋진 사람이 되고 싶어요

**Comment by KORabcd** [comment]
비밀 댓글입니다.

**Reply by dongkyu12345** [reply]
비밀 댓글입니다.

**Comment by hoo161206** [comment]
안기모취한 윤시원씨
